astro/src/content/blog/fusionauth-web-summit.mdx

```diff
@@ -0,0 +1,392 @@
+
+**Stay on Time, do not go over**
+
+When I have hosted PITCH, if you went over, I’d simply remove that time from your questions. Not all hosts will do this. Some will cut you off mid-sentence not long after the 3 minute mark. There’s a countdown timer on screens in front of the stage you will be able to see.
+
+**Prepare for the Questions**
+
+Prepare as much for the questions as you prepare for the answers. A series of great succinct answers will help your chances a lot.
+
+### **Engaging with Startups and Companies**
+
+When engaging with startups, consider how their innovations could impact your industry. Look for potential collaborations or investment opportunities.
+
+Don’t sell to booth owners, they have paid good money as they are selling their stuff. Sure, drop by and say hi and drop a card, but don’t abuse their time.
+
+### **Identifying Trends and Innovations**
+
+Pay attention to recurring themes and emerging trends. These insights can provide a competitive edge in your work or business.
+
+At the 2022 Web Summit, there were a lot of mumbles about the future of AI. It wasn’t loud but it was there and recurring.
+
+ChatGPT launched about 2 weeks later. The 2023 Summit was all about AI.
+
+### **Approaching the Media**
+
+We asked Mike Butcher, Techcrunch Editor-at-Large and Web Summit veteran for his tips on engaging with the media. So here they are:
+
+1. Pinpoint your journalist before the event with an email/contact to set-up a meeting or offer them an story exclusive so the story comes out whilst your are there
+
+2. Scour the agenda to see where a journalist is moderating a panel/interview and try and grab them as they leave the stage
+
+3. “Goal hang” where the media enter the main venue from the VIP/Press area
+
+4. Repeat steps 2 & 3
+
+Make sure you follow Mike - @mikebutcher and check out [The Europas Awards](https://www.theeuropas.com/) and event he created which runs during Web Summit.
+
+![Enter image alt description](/img/blogs/fusionauth-web-summit/mike-butcher-wide.jpg)
+
+## **Making the Most of Social Events and Parties**
+
+### **Side Events**
+
+As happens with most events of such a size, there are a ton of non-official but amazing side events every night (and some early morning ones too). Some of these events are extremely niche but you just might find your tribe there.
+
+Luma published a list of them [here](https://lu.ma/websummit) last year, but keep an eye on Meetup and Eventbrite too.
+
+Side events tend to fill up super quickly as people tend to register for everything, and choose what they attend closer to the time. Most experienced side event organizers know this, so if there’s a waitlist, join it as you’ll probably get in.
+
+The larger "official" parties will be overrun, try to go for the smaller, themed events that are in your area of interest. This will alow you to find people that are potential valuable connections.
```

Comment:
```suggestion
The larger "official" parties will be overrun so go for the smaller, themed events that are in your area of interest. This will allow you to find people that are potential valuable connections.
```